tests/unit/test_space.py

```diff
@@ -370,11 +407,31 @@ def test_box_contains_point(point: tf.Tensor) -> None:
 )
 def test_box_does_not_contain_point(point: tf.Tensor) -> None:
     assert point not in Box(tf.constant([-1.0, 0.0, -2.0]), tf.constant([2.0, 1.0, -0.5]))
+    assert not Box(tf.constant([-1.0, 0.0, -2.0]), tf.constant([2.0, 1.0, -0.5])).contains(point)
+
+
+@pytest.mark.parametrize(
+    "points, contains",
+    [
+        (tf.constant([[-1.0, 0.0, -2.0], [-1.1, 0.0, -2.0]]), tf.constant([True, False])),
+        (tf.constant([[[0.5, 0.5, -1.5]]]), tf.constant([[True]])),
+    ],
+)
+def test_box_contains_broadcasts(points: tf.Tensor, contains: tf.Tensor) -> None:
+    box = Box(tf.constant([-1.0, 0.0, -2.0]), tf.constant([2.0, 1.0, -0.5]))
+    npt.assert_array_equal(contains, box.contains(points))
+    # point in space raises (because python insists on a bool)
+    with pytest.raises(TF_DEBUGGING_ERROR_TYPES):
+        _ = points in box
```

Comment:
Is this just to test that using `in` on a tensor of points raises an error?

Reply:
Yes